examples/pomtospdx/README.md

```diff
@@ -0,0 +1,13 @@
+# Plugin Example: Parse a pom.xml file and generate an SPDX 2.3 JSON document
+
+## Writing the Plugin
+
+In this example, we will parse an example pom.xml file and create an SPDX document with the data. The plugin is called `mvnpom`. We use an existing pom.xml parser to get the data. We have restricted parsing to only SPDX 2.3 as the SPDX version and JSON as the data format, but other versions and formats can be supported.
+
+The required function to implement for the plugin is the `GetSpdxDocument` method. This must return an object of type `AnyDocument`.
```

Comment:
If you think about it, the plugin's job is not to return the document. The plugin should only return a data structure representing the data it knows about (probably a package representing the code and its dependencies). That structure may be part of a document with more than one code repository, for example.